### PR TITLE
Manual Lat/Lon entry improvements

### PR DESCRIPTION
**Pull request by landswellsong**
- Map is now on by default, `useMap` state removed, manual lat/lon entry boxes are always shown.
- `polygon` is not passed to the search if nothing is selected and manual lat/lon input select the full map.
- default bounds changed to full map.
- manual lat/lon act an extra layer of selection except if the whole map is selected. this is subject to change (#238), currently let's leave this behaviour as is not to rework the API.
- if anything but the full map is selected in the manual lat/lon input the map would show the appropriate selection (distinct from actual selection objects).
- Top panel (`Well`) removed.

### CODE REVIEW

File: frontend/app/components/SearchForm.js [hunk 1]
```diff
@@ -32,33 +32,27 @@ class SearchTrigger extends Component {
...
-                        )
-                    })
-                )
-            });
-
-            /* flush possible selection */
-            this.props.selected.clearSelection();
+        /* add selection if any */
+        selection = this.props.selection.elements.length > 0 ?
+            this.props.selection.elements.slice() : [];
```
Comment (0181532686cf4a31163be0bf3e6bb6732bf) [inline frontend/app/components/SearchForm.js:40]: no need to create a copy here

**Comment by 0181532686cf4a31163be0bf3e6bb6732bf** [comment]
It's impossible to revert default selection back to the initial state (only manually)

**Review by 0181532686cf4a31163be0bf3e6bb6732bf — CHANGES_REQUESTED** [review]
1. Property copying inside search widget
2. No way to restore default selection back other than by hand

**Comment by landswellsong** [comment]
1. We need a copy because multiple `getSessions()` calls without a state update would otherwise grow `this.props.selection` array indefinitely. We will hardly have more than a dozen items there and they are passed as reference inside the array anyway. What is wrong with copying?
2. Future task unrelated to this PR (#239)

**Comment by 0181532686cf4a31163be0bf3e6bb6732bf** [comment]
1. `selection` component of the state is unaffected by `getSessions` call. It's only managed from map/selection utils. Copying is just not needed there
2. why it's unrelated?

**Comment by landswellsong** [comment]
1. Later on [I alter this array](https://github.com/IonosatMicro/promis/pull/241/files/0655ea431fbb847e1123e243a93cf1e9cd1943b4#diff-55411a068470688eac6b4b6e456dcbbcR47). I need it copied so it doesn't `push` a new one every time. 
2. There was a request to try several variants of this functionality. Currently we are fixing issues to merge a PR based on the feedback, not implementing new features.

**Comment by 0181532686cf4a31163be0bf3e6bb6732bf** [comment]
Ok

**Comment by landswellsong** [comment]
Thx

**Pull request merged**